frontend/src/main/scala/bloop/dap/DebuggeeRunner.scala

```diff
@@ -46,7 +46,7 @@ private final class MainClassDebugAdapter(
       env,
       workingDir,
       mainClass.`class`,
-      mainClass.arguments.toArray,
+      (mainClass.arguments ++ mainClass.jvmOptions).toArray,
```

Comment:
Hmmm, interesting. I thought we were passing the JVM options. Did you double-check that the test passes without this fix?

Reply:
yep, it fails without it.